### ISSUE TITLE
Icon code doesn't match Customized Code [How to make copy-paste-able icons for Photoshop]

### ISSUE THREAD

**Issue by raduamarie** [fontello/fontello#381]
Hi all!

We really like fontello, and one of the problems we encountered is that if we wanted to also use the icons in photoshop for prototyping, the icons would not correspond (if we select the icon and paste it in photoshop it would not be the same icon). We figured the problem was that the actual icon would not correspond to the same "Custom Code" from the above input field, so we came up with a quick dirty fix, modifying the Codes with the actual icon codes.

`$('.font-fontello').each(function(index, value){
	$(this).parent().find('input.char-editable').val($(this).text()).trigger("change");
});`

![screen shot 2015-01-25 at 13 33 52](https://cloud.githubusercontent.com/assets/2953026/5891326/3bab4480-a497-11e4-8b8c-42d0ce53ebf7.png)

Now if you select the icon, it will correspond to the actual icon in Photoshop, another nice feature to have is in the demo.html that fontello outputs to be able to double-click, select and copy the icons :)

Thanks! :)

**Comment by raduamarie** [comment]
A quick dirty fix to the `demo.html` page to make it selectable and usable is the current script that should go in footer.php.

```html
<script src='https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js'></script>
<script>
	$( "<style>i:before { display: none!important; }</style>" ).appendTo( "head" );

	var styles = {      
		fontFamily: "fontello",
		fontStyle: "normal",
		fontWeight: "normal"   
	};

	$('i').each(function(){
   	 $(this).html(window.getComputedStyle(this,':before').content.replace(/\"/g, "")).css(styles);
	});
</script>
```

**Comment by paul2moro** [comment]
This is awesome! Even if it's a quick fix, at least I (as a designer) am now able to quickly copy icons from the demo page and paste them into Photoshop. thanks so much!

**Comment by puzrin** [comment]
@raduamarie Do you mean, that if you customize codes, those are not saved? That's strange.

Could you check content of CSS files? Real codes are there (in other case demo could not show those).

**Comment by raduamarie** [comment]
No, if you customize the icons, they are changed, but the issue is that the actual Displayed Icon code != Customized Code, so that if you double click the icon to select it, it's not the same code. The javascript I shared was so that all the codes are equal to their actual icons :)

For copy-paste-ability :)

In other words, if I go to Customized Codes, and double click my previously selected smiley face, download the font, enter photoshop and paste the smiley face, photoshop doesn't recognize the smiley face, because the codes are not the same :)

**Comment by puzrin** [comment]
Do you try to copypaste directly from fontello site instead of demo page? That should not work, end never expected to work.

**Comment by raduamarie** [comment]
Yeah :)

On the demo page the icon is not copyable, because it's an ":before" element, that's why I shared my 2nd script. 

To make a even more simple explanation, we need to make some prototyping in Photoshop, and would be awesome to use the icons from fontello in Photoshop as well, the perfect example is http://fontawesome.io/cheatsheet/ on that page you can double-click the icon and paste it in Photoshop and the icon corresponds in both places: the demo page and Photoshop. This doesn't happen in fontello :(

A way to reproduce is to load the fontello website, select all the icons, download the pack, install the font in Photoshop, go to the demo page, choose an icon, grab their ":before" content, and paste it in Photoshop, and in Photoshop the icon that you pasted would not be the same one the from where you copied.

**Comment by puzrin** [comment]
If icons from demo page become copy-pasteable, will it solve your problem? It's not a big problem to do. Just nobody asked before.

**Comment by paul2moro** [comment]
hey @puzrin, yes, allowing the icons to be easily selectable & copiable would be great, especially for us designers.

also, different issue but I think the Customise Names & Customise Codes could be one and the same page :)

**Comment by raduamarie** [comment]
Yup, making it copy-paste-able would be awesome :)

**Comment by puzrin** [comment]
Closed in flavour of #382

**Comment by puzrin** [comment]
copypaste should be fixed now, sorry for delay
